examples/vue-vanilla-custom-dist/client/build/client/.vite/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
Was this directory supposed to be under `.gitignore`?

Reply:
Yes, good catch!